components/monitoring/README.md

```diff
@@ -0,0 +1,113 @@
+###On boarding to services to Monitoring
+
+#1. Metrics exporter services
+a. The intended service should export the metrics from the application so that prometheus is able to understand it. 
```

Comment:
This list got all run together into one line, which makes it hard to read, so use the format in https://www.markdownguide.org/basic-syntax/#lists-1  instead of a. b. c.